app/models/reaction.rb

```diff
@@ -60,7 +60,7 @@ def for_display(user)
     def cached_any_reactions_for?(reactable, user, category)
       class_name = reactable.class.name == "ArticleDecorator" ? "Article" : reactable.class.name
       cache_name = "any_reactions_for-#{class_name}-#{reactable.id}-#{user.updated_at}-#{category}"
```

Comment:
@rhymes [introduced me to the datetime format `.rfc3339`](https://github.com/thepracticaldev/dev.to/pull/4812/files#diff-4676c008b11a5480d73d4a6de01e45b9R295). Can you apply that here to user.updated_at so we end up with better key formats in Redis? Please :) 

Will be interested to see how many keys this makes, it feels like since the where clause is simply a presence check we might not even need the cache.

Reply:
Done